documentation/docs/glossary.md

```diff
@@ -7,103 +7,102 @@ keywords:
 - IOTA
 - verification method
 - verifiable data registry
+- reference
 ---
 
 # Glossary
 
-The first part describes the terminology by the W3C.
-The second part describes the terminology for IOTA related topics.
+This section defines the terms used in this specification, (sourced from W3)[https://www.w3.org/TR/did-core/#terminology], and throughout decentralized identifier infrastructure. A link to these terms is included whenever they appear in this specification.
 
-[Source](https://www.w3.org/TR/did-core/#terminology)
+The first part of the glossary describes the terminology by the W3C. The second describes the terminology for IOTA-related topics.
```

Comment:
```suggestion
The first part of the glossary describes the terminology by the W3C. The second describes the terminology for IOTA-related topics. 
```
Perhaps we could add both section separators as headers, and add a link here. wdyt?

Reply:
Doable👍